core/geometry/src/curve/Arc3d.ts

```diff
@@ -445,6 +446,36 @@ export class Arc3d extends CurvePrimitive implements BeJSONFunctions {
     }
     return LineSegment3d.create(start, end);
   }
+  /**
+   * Create a circular arc defined by start and end points and radius.
```

Comment:
Because an arc cannot be defined uniquely bt start, end, and radius I will mention the helper point too.

Reply:
Eh, it's a small detail. We want that first sentence to be short.